src/metalnx-web/src/main/resources/static/js/collection.js

```diff
@@ -146,16 +146,26 @@ function deleteInfoAction(path){
 
 	var paths = [];
 	paths.push(path);
-	var url = "/metalnx/fileOperation/delete/";
+	var url = "/metalnx/fileOperation/deleteNoRedirect/";
 
 	ajaxEncapsulation(
 			url,
 			"POST",
 			{paths: paths},
-			function (data) {
+			function (failedDeletions) {
 				unsetOperationInProgress();
 				$('#actionsWait').hide();
-				$('#deleteConfirmationModal').modal();
+				
+				// The delete operation failed if the array is not empty.
+				// The failure is likely permissions related.
+				if (failedDeletions.length > 0) {
+					//toastr.error("Failed! The collection was not deleted."); // This works!
```

Comment:
should this be uncommented?

Reply:
It is a different way to display the failure message, but I didn't use it because it didn't fit the theme.

Will remove.

Reply:
done.